ddp-workspace/projects/ddp-sdk/src/lib/services/activity/activityQuestionConverter.service.ts

```diff
@@ -278,8 +278,8 @@ export class ActivityQuestionConverter {
         numericBlock.placeholder = questionJson.placeholderText;
         const intRangeValidation = questionJson.validations.find(validation => validation.rule === ValidationRuleType.IntRange);
         if (intRangeValidation) {
-            numericBlock.min = intRangeValidation.min || null;
-            numericBlock.max = intRangeValidation.max || null;
+            numericBlock.min = isNaN(intRangeValidation.min) ? null : intRangeValidation.min;
+            numericBlock.max = isNaN(intRangeValidation.max) ? null : intRangeValidation.max;
```

Comment:
The idea is correct.
There is one note only. 
Our API spec says that `min` and `max` can be null (not sure if it is the valid case)
![ed](https://user-images.githubusercontent.com/7396837/182214814-5e21edf1-6e8e-4f25-91e8-e56f07cd3488.png)

So, more general would be to check whether it is not null and not undefined, 
e.g. like 
`numericBlock.min = intRangeValidation.min == null ? null : intRangeValidation.min;`

Reply:
that's a good point. Null check added already